### PR TITLE
Hour command

### PR DESCRIPTION
**Pull request by petrickjerico**
### Changes

|Description|Picture|
|-|-|
|Added `/hour` command: simplify submission into just duration of use (in hours). Type check ensures that hours are in `float`.|Screenshot 2023-07-19 at 9 32 31 PM|
|Bot messages are also customized to indicate that a user has submitted using a particular command.|Screenshot 2023-07-19 at 9 32 01 PM|
|[Form](https://docs.google.com/forms/d/1wkzfpP4LrVAabw23GwXFU73vMvNWrbbqlnRidCa3kJQ/edit) updated accordingly.|Screenshot 2023-07-19 at 9 43 43 PM|

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -90,6 +91,23 @@ async def abort_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
     else:
         await update.message.reply_text("You don't have an active session. Use /on to start a new timer.")
 
+async def hour_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    user_name = update.effective_user.username
+    if USER_NAME_MAPPING.get(user_name, "Unknown") == "Unknown":
+        await update.message.reply_text("You are not registered yet. Contact @samtjong to register before you can use this bot.")
+    elif len(context.args) != 1 or not context.args[0].replace('.','',1).isdigit():
+        # Check if too many argoments or argument is not a number (int or float)
```
Comment (samtjong23) [inline bot.py]: Typo 'argoments'

File: bot.py [hunk 2]
```diff
@@ -90,6 +91,23 @@ async def abort_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
     else:
         await update.message.reply_text("You don't have an active session. Use /on to start a new timer.")
 
+async def hour_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    user_name = update.effective_user.username
+    if USER_NAME_MAPPING.get(user_name, "Unknown") == "Unknown":
```
Comment (samtjong23) [inline bot.py]: Standardize to user_name not in USER_NAME_MAPPING

File: bot.py (hunk 1 above)
Comment (petrickjerico) [reply bot.py]: [Resolved!](https://github.com/samtjong23/airconTeleBot/pull/3/commits/e32b41458c00a49617a249df0e68b31d45b0096f)

File: bot.py (hunk 2 above)
Comment (petrickjerico) [reply bot.py]: [Resolved!](https://github.com/samtjong23/airconTeleBot/pull/3/commits/acd25ca16c9fa5596825f13fb850daca1873e381)

**Pull request merged**